tools/lib/run-cmake-configure.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+script_dir="$(dirname "$(readlink -f "$0")")"
+
+cmake_args=(
+    -B "${build_dir}"
+    -S "${script_dir}/../.."
+    -G "${generator}"
+    -DCMAKE_PREFIX_PATH="${qt_base}/${qt_version}/${qt_arch}"
+    -DCMAKE_PROJECT_TOP_LEVEL_INCLUDES="${script_dir}/../../external/cmake-conan/conan_provider.cmake"
+    -DCONAN_HOST_PROFILE="${conan_profile}"
+    -DCONAN_BUILD_PROFILE="${conan_profile}"
+    -DSCWX_VIRTUAL_ENV="${venv_path}"
```

Comment:
If I am remembering correctly, `clangd` uses `compile_commands.json` to find imports. It may be worth making sure that compile commands are generated. (They seem to be for me, but if that is not configured someplace, this may be worth adding).
```suggestion
    -DSCWX_VIRTUAL_ENV="${venv_path}"
    -DCMAKE_EXPORT_COMPILE_COMMANDS=on
```

Reply:
Done